listener/listener_config.toml

```diff
@@ -25,7 +25,7 @@ LogFile  = "__screen__"
 
 [Listener]
 bind = "http://localhost:1947"
-max_work_order_count = 10
+max_work_order_count = 1000
```

Comment:
Add description of this parameter in the config.

Reply:
Done.